### PR TITLE
Add issue marker noting guidance about decoy values will be resolved in CR.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to note that issue #150 will be addressed in CR, with more direct involvement by PING (and other concerned parties), by performing a more detailed analysis on decoy values.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-bitstring-status-list/pull/171.html" title="Last updated on May 14, 2024, 1:02 PM UTC (eee3c71)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-bitstring-status-list/171/b571224...eee3c71.html" title="Last updated on May 14, 2024, 1:02 PM UTC (eee3c71)">Diff</a>

### CODE REVIEW

**Review by KDean-Dolphin — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by David-Chadwick — APPROVED** [review]

**Review by iherman — APPROVED** [review]

**Review by npdoty — APPROVED** [review]
This accurately captures the status of exploration of the technique. Thanks.

**Comment by msporny** [comment]
@npdoty wrote:
> This accurately captures the status of exploration of the technique. Thanks.

Great, thanks for the confirmation, @npdoty; we'll get this merged as soon as we can and include it in the CR-ready draft. 

In an attempt to parallelize the approvals we need, could you please convey that you are ok w/ proceeding to CR as long as the text in this PR is included:

https://github.com/w3c/transitions/issues/603#issuecomment-2104642820

**Review by longpd — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Review by decentralgabe — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-05-15-vcwg#section3-1) on 2024-05-15

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.1. Add issue marker noting guidance about decoy values will be resolved in CR. (pr vc-bitstring-status-list#171)

_See github pull request [vc-bitstring-status-list#171](https://github.com/w3c/vc-bitstring-status-list/pull/171)._




**Manu Sporny:** If we can merge this today, we can include it in the snapshot for publication.  

> *Manu Sporny:* I will merge and integrate after this meeting.

**Brent Zundel:** You have approval to merge this after this call.  

</details>

**Comment by brentzundel** [comment]
Merging, as discussed on the call.

**Pull request merged**
